cms/db/base.py

```diff
@@ -22,15 +22,15 @@
 import ipaddress
 from datetime import datetime, timedelta
 
+from sqlalchemy.dialects.postgresql import ARRAY, CIDR, JSONB, OID
```

Comment:
Nit: typo in commit title (incresse).

Reply:
Also, isn't there some other place where ordering matters (I'm thinking cms/cms.log). Please double check.

Reply:
It would be nice to have a command line tool to enforce/check this as we go on. Maybe pycharm uses something that we can use as well in a precommit hook?

Reply:
Will fix the commit when rebasing.

`cms/log.py` is correctly imported by `cms/__init__.py`. I did the following check: I looked for comments that were close to the lines affected by the commit and made sure that none of them said "needs to be first", "do not remove", etc. This showed that in fact PyCharm had removed an unused import, `cmsranking.Logger`, that we needed. So I readded it.

I don't know if we can use the PyCharm operation from the command line. And I don't know of other tools that do it.